src/services/languages.js

```
@@ -0,0 +1,51 @@
+const Language = require('~/models/languages')
+
+const getLanguages = async () => {
+  return await Language.find()
+}
+
+const getLanguageById = async (id) => {
+  return await Language.findById(id)
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)